app/main.py

```diff
@@ -1,2 +1,44 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
```

Comment:
load factor is a constant - you can set it as a class attr + you can use Fraction type to store value